modules/firebase/utils.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  collection,
+  type CollectionReference,
+  type Firestore,
+} from 'firebase/firestore'
+import { getApp, initializeApp } from 'firebase-admin/app'
+
+export const getAdminApp = () => {
+  try {
+    return getApp('adminApp')
+  } catch (error) {
+    return initializeApp({}, 'adminApp')
+  }
+}
+
+export interface Organization {
```

Comment:
We should setup an issue that we eventually re-use the types from the Firebase backend setup.

Reply:
Definitely, for now I just copy pasted it as quick solution.

Reply:
https://github.com/StanfordBDHG/ENGAGE-HF-Web-Frontend/issues/15

Reply:
Thank you!